jwst/ami/oifits.py

```diff
@@ -235,381 +158,258 @@ def populate_nrm_dict(self):
 
         sta_index = np.arange(N_ap) + 1
 
-        pscale = info4oif.pscale_mas/1000.  # arcsec
-        isz = self.ff.scidata.shape[1]  # Size of the image to extract NRM data
+        pscale = instrument_data.pscale_mas / 1000.0  # arcsec
+        isz = self.fringe_fitter.scidata.shape[
+            1
+        ]  # Size of the image to extract NRM data
         fov = [pscale * isz] * N_ap
-        fovtype = ['RADIUS'] * N_ap
-
-        self.oifits_dct = {
-            'OI_VIS2': {'VIS2DATA': self.vis2,
-                       'VIS2ERR': self.e_vis2,
-                       'UCOORD': ucoord,
-                       'VCOORD': vcoord,
-                       'STA_INDEX': self._format_STAINDEX_V2(self.bholes),
-                       'MJD': t.mjd,
-                       'INT_TIME': info4oif.itime,
-                       'TIME': 0,
-                       'TARGET_ID': 1,
-                       'FLAG': flagVis,
-                       'BL': bl_vis
-                       },
-
-           'OI_VIS': {'TARGET_ID': 1,
-                      'TIME': 0,
-                      'MJD': t.mjd,
-                      'INT_TIME': info4oif.itime,
-                      'VISAMP': self.visamp,
-                      'VISAMPERR': self.e_visamp,
-                      'VISPHI': self.visphi,
-                      'VISPHIERR': self.e_visphi,
-                      'UCOORD': ucoord,
-                      'VCOORD': vcoord,
-                      'STA_INDEX': self._format_STAINDEX_V2(self.bholes),
-                      'FLAG': flagVis,
-                      'BL': bl_vis
-                      },
-
-           'OI_T3': {'TARGET_ID': 1,
-                     'TIME': 0,
-                     'MJD': t.mjd,
-                     'INT_TIME': info4oif.itime,
-                     'T3PHI': self.cp,
-                     'T3PHIERR': self.e_cp,
-                     'T3AMP': self.camp,
-                     'T3AMPERR': self.e_camp,
-                     'U1COORD': u1coord,
-                     'V1COORD': v1coord,
-                     'U2COORD': u2coord,
-                     'V2COORD': v2coord,
-                     'STA_INDEX': self._format_STAINDEX_T3(self.tholes),
-                     'FLAG': flagT3,
-                     'BL': bl_cp
```

Comment:
This (and the other uses of `BL`) are unused (`populate_oimodel` does not propagate them to the model. I removed `BL` here but wanted to call it out as it may be a bug that these are not added to the model.

Reply:
Summarizing conversation on Slack, it is fine to remove unused BL columns since they don't make it into the final OIFITS (ami-oi.fits) files; if needed in the future they can be added back in.